webook/calendar_buddy/contexts/fullcalendar/translator.py

```diff
@@ -0,0 +1,97 @@
+from ....calendar_buddy.mediative.mediative_event import MediativeEvent
+from ....calendar_buddy.mediative.mediative_resource import MediativeResource
+
+from ....calendar_buddy.contexts.fullcalendar.models import Event, Resource
+
+
+def _set_defaults_on_dict(dict_to_set_defaults_on: dict(), defaults_to_set: dict()) -> dict:
```

Comment:
dont use `dict()` it is how you create a dict literally, there are multiple ones like this, please change them all. see previous comment

Reply:
I wonder if your reinventing default dict from the collections module. Lets talk about what is going on here later and see if there might be some other way of doing this.